File: mostFrequentChar.js
const mostFrequentChar = (s) => {
  const obj = [...s].reduce((a,c) => (a[c] = ++a[c] || 1, a), {})
  
  return [...s].reduce((a,c) => obj[c] > obj[a] ? c : a, s[0])
};

File: mostFrequentChar.py
def most_frequent_char(s):
  count = {}

  for c in s:
    if c not in count:
      count[c] = 0

    count[c] += 1

    max = 0
    char = ''

  for key in count:
    if max < count[key]:
      max = count[key]
      char = key


  return char